project.clj

```diff
@@ -119,6 +119,9 @@
   ;; code that we have.
   :classifiers  [["test" :testutils]]
 
+  ;; Inovke "lein test :all" to run all tests
+  :test-selectors {:default (fn [m] (not (:slow m)))}
```

Comment:
Hopefully this is what you meant: https://github.com/puppetlabs/pe-puppetdb-extensions/pull/166